### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by MihlaliMgwebi**

### CODE REVIEW

File: github-by-mish/src/component/Error/NotFound.tsx [hunk 1]
```diff
@@ -10,8 +10,9 @@ const NotFound = () => {
         </Header>
         <Content>
           <div className='mt-16 flex-auto'>
-            <section className='bg-black p-12 min-h-[70rem]'>
-              <div className='w-100 mx-auto flex justify-center items-center text-center lg:text-left min-h-[63rem]'>
+            {/* <section className='bg-black p-12 min-h-[70rem]'> */}
```
Comment (theadley) [inline github-by-mish/src/component/Error/NotFound.tsx:13]: 👀

**Review by theadley — CHANGES_REQUESTED** [review]
One line of commented-out code

File: github-by-mish/src/component/Error/NotFound.tsx (hunk 1 above)
Comment (MihlaliMgwebi) [reply github-by-mish/src/component/Error/NotFound.tsx:13]: Oh goodness me. Removed the dead code.

**Review by theadley — APPROVED** [review]
Good thanks!

**Pull request merged**